### ISSUE TITLE
[mqtt] logs on start «No config description found for URI 'thing-type:mqtt:string_channel'»

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-addons#13772]
I have an MQTT-Broker thing defined in Main UI, and MQTT-Things defined both in Main UI and in `.things` files.  When openHAB 3.3 boots it prints:
```
2022-11-24 10:51:04.859 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading model 'system_started.rules'
2022-11-24 10:51:15.104 [INFO ] [e.automation.internal.RuleEngineImpl] - Rule engine started.
2022-11-24 10:51:15.559 [INFO ] [org.openhab.ui.internal.UIService   ] - Started UI on port 8080
2022-11-24 10:51:20.603 [INFO ] [rulesupport.loader.ScriptFileWatcher] - Loading script '/etc/openhab/automation/js/pompi.js'           2022-11-24 10:51:24.888 [INFO ] [.transport.mqtt.MqttBrokerConnection] - Starting MQTT broker connection to '127.0.0.1' with clientid openhab_client               
2022-11-24 10:51:28.001 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:string_channel'
2022-11-24 10:51:28.006 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:string_channel'
2022-11-24 10:51:28.076 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:dimmer_channel'
2022-11-24 10:51:28.080 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:dimmer_channel'
2022-11-24 10:51:28.092 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:string_channel' 
2022-11-24 10:51:28.080 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:dimmer_channel' 
2022-11-24 10:51:28.092 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:string_channel' 
2022-11-24 10:51:28.096 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:string_channel'
2022-11-24 10:51:28.110 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:string_channel'
2022-11-24 10:51:28.114 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:string_channel'
2022-11-24 10:51:28.146 [WARN ] [ation.ConfigDescriptionValidatorImpl] - No config description found for URI 'thing-type:mqtt:dimmer_channel'
2022-11-24 10:51:28.150 [WARN ] [ation.ConfigDescriptionValidatorImpl] - Skipping config description validation because no config description found for URI 'thing-type:mqtt:dimmer_channel'
```
I have no thing or item called literally `thing-type:mqtt:dimmer_channel`.

What shall I do to avoid the above warnings to appear in the log, apart from disabling logging completely?

**Comment by VanDeStouwe** [comment]
I have exactly the same issue

**Comment by martingrassl** [comment]
I get this issue since update to 3.4

**Comment by crazyelectron-io** [comment]
+1, but not for all MQTT Things (although they follow the exact same format).

**Comment by J-N-K** [comment]
Correct solution: make sure that the config description is in the registry. The binding should take care of that.

**Comment by crazyelectron-io** [comment]
Thanks for your quick reply @J-N-K but I'm not sure I understand what you mean. Which _registry_ are you referring to?

**Comment by J-N-K** [comment]
Fixed in 4.0 with https://github.com/openhab/openhab-core/pull/3330. The issue is that the `ThingManager` tried to initialize the thing before the binding is fully loaded. This will no longer happen in OH4.
